src/components/contextual/pages/pool/PoolBalancesCard/BoostedPool.vue

```diff
@@ -39,6 +41,17 @@ function getUnderlyingTokens(address: string) {
     ? [linearPools[address].mainToken, linearPools[address].wrappedToken]
     : [];
 }
+
+function getTokenShare(address: string) {
+  const totalSupply = props.pool.onchain.totalSupply;
```

Comment:
I suppose this total supply applies to all the linear pools (bb-a-USDT/AUSDC/aDAI)